fido/fido.py

```diff
@@ -255,7 +254,6 @@ def fetch(
     body='',
     timeout=DEFAULT_TIMEOUT,
     connect_timeout=DEFAULT_CONNECT_TIMEOUT,
-    content_type=DEFAULT_CONTENT_TYPE,
```

Comment:
Thanks for the review! 
1) I was going to bump for sure, just putting out the change first :)
2) Another option could be to leave the default argument 'content-type=None' in that function to preserve the contract but make it a NO-OP. I'd prefer bumping and removing it though, for the sake of cleaner code